ts/state/ducks/metaGroups.ts

```diff
@@ -680,7 +680,7 @@ async function handleMemberAddedFromUI({
     }
   } catch (e) {
     window.log.warn(
-      'handleNameChangeFromUIOrNot: pushChangesToGroupSwarmIfNeeded failed with:',
+      'handleMemberAddedFromUIOrNot: pushChangesToGroupSwarmIfNeeded failed with:',
```

Comment:
Function is actually called `handleMemberAddedFromUI`? Do we care?

Reply:
fixed!